bundles/CoreBundle/Migrations/Version20211103055110.php

```diff
@@ -50,7 +50,7 @@ public function down(Schema $schema): void
     {
         $db = \Pimcore\Db::get();
 
-        $classes = $db->fetchRow('SELECT id FROM classes');
+        $classes = $db->fetchAssociative('SELECT id FROM classes');
```

Comment:
That's unfortunalely not returning the same result. 
`fetchAssociative()` is equal to `fetchAll()` but `fetchRow()` is only returning the first row using indices.